crates/uv/src/commands/python/install.rs

```diff
@@ -41,11 +44,19 @@ pub(crate) async fn install(
 
     let targets = targets.into_iter().collect::<BTreeSet<_>>();
     let requests: Vec<_> = if targets.is_empty() {
-        if let Some(requests) = requests_from_version_file().await? {
-            requests
+        // Read from the version file, unless `isolated` was requested
+        let version_file_requests = if isolated {
+            if PathBuf::from(".python-version").exists() {
+                debug!("Ignoring `.python-version` file due to isolated mode");
+            }
+            if PathBuf::from(".python-versions").exists() {
```

Comment:
Some implicit coupling here between `requests_from_version_file` and the paths you're checking... Maybe just make shared `PYTHON_VERSION` and `PYTHON_VERSIONS` constants and reference them in both places, so the connection is clear in case we change them later?

Reply:
I could do that, will need in `pin` too.